src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -2460,18 +2445,6 @@ void InitSolarHeatGains(EnergyPlusData &state)
 
     auto &Surface = state.dataSurface->Surface;
 
-    // Using/Aliasing
-    using DaylightingDevices::TransTDD;
-    using General::POLYF;
-    using SolarShading::CalcInteriorSolarDistribution;
-    using namespace DataWindowEquivalentLayer;
-    using SolarShading::SurfaceScheduledSolarInc;
-    using SolarShading::WindowScheduledSolarAbs;
-
-    auto &AbsDiffWin = state.dataHeatBalSurfMgr->AbsDiffWin;
-    auto &AbsDiffWinGnd = state.dataHeatBalSurfMgr->AbsDiffWinGnd;
-    auto &AbsDiffWinSky = state.dataHeatBalSurfMgr->AbsDiffWinSky;
```

Comment:
[src/EnergyPlus/HeatBalanceSurfaceManager.cc:2471]:(style),[variableScope],The scope of the variable 'AbsDiffWin' can be reduced.
[src/EnergyPlus/HeatBalanceSurfaceManager.cc:2472]:(style),[variableScope],The scope of the variable 'AbsDiffWinGnd' can be reduced.
[src/EnergyPlus/HeatBalanceSurfaceManager.cc:2473]:(style),[variableScope],The scope of the variable 'AbsDiffWinSky' can be reduced.